modules/data.atmosphere/R/met.process.R

```diff
@@ -59,7 +59,12 @@ met.process <- function(site, input_met, start_date, end_date, model,
   }
   
   # set up connection and host information
-  con <- db.open(dbparms)
+  bety <- src_postgres(dbname   = dbparms$dbname, 
```

Comment:
Will this need to be  ```dplyr::src_postgres()```? You'll also probably need to add dplyr to the DESCRIPTION file in data.atmosphere under Imports.